### PR TITLE
Add a step on enabling automatic etcd monitoring updates

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@bergerhoffer Could you PTAL and merge? Thanks.

File: install_config/monitoring/configuring-etcd-monitoring.adoc [hunk 1]
```diff
@@ -185,3 +185,13 @@ $ oc apply -f etcd-cert-secret.yaml
...
+
+. If you want `etcd` monitoring to be automatically updated when you update {product-title}, set this variable in the Ansible inventory file to `true`:
++
+`openshift_cluster_monitoring_operator_etcd_enabled`
++
+If you run `etcd` on separate hosts, specify the nodes by IP addresses using this Ansible variable:
++
+`openshift_cluster_monitoring_operator_etcd_hosts`
++
+If the IP addresses of the `etcd` nodes change, you need to update this list.
```
Comment (bergerhoffer) [inline install_config/monitoring/configuring-etcd-monitoring.adoc:197]: need to -> must

File: install_config/monitoring/configuring-etcd-monitoring.adoc [hunk 2]
```diff
@@ -185,3 +185,13 @@ $ oc apply -f etcd-cert-secret.yaml
 . Now that you have configured authentication, visit the *Targets* page of the web interface again. Verify that `etcd` is now being correctly monitored. It might take several minutes for changes to take effect.
 +
 image::etcd-monitoring-working.png[]
+
+. If you want `etcd` monitoring to be automatically updated when you update {product-title}, set this variable in the Ansible inventory file to `true`:
++
+`openshift_cluster_monitoring_operator_etcd_enabled`
```
Comment (bergerhoffer) [inline install_config/monitoring/configuring-etcd-monitoring.adoc:191]: Not critical, but this variable looks a little odd on its own line like this. I'd suggest to put it in a code block, like:

```
----
openshift_cluster_monitoring_operator_etcd_enabled=true
----
```

File: install_config/monitoring/configuring-etcd-monitoring.adoc [hunk 3]
```diff
@@ -185,3 +185,13 @@ $ oc apply -f etcd-cert-secret.yaml
...
++
+If you run `etcd` on separate hosts, specify the nodes by IP addresses using this Ansible variable:
++
+`openshift_cluster_monitoring_operator_etcd_hosts`
```
Comment (bergerhoffer) [inline install_config/monitoring/configuring-etcd-monitoring.adoc:195]: Same comment about the above, maybe with an example of how the IP addresses look

```
----
openshift_cluster_monitoring_operator_etcd_hosts= // i don't know how they need to be... comma separated? in quotes?
----
```

**Review by bergerhoffer — COMMENTED** [review]
Just some thoughts

**Comment by rh-max** [comment]
@bergerhoffer Thanks! Implemented. If you have no further feedback, this is ready for merging.

**Comment by bergerhoffer** [comment]
Sorry for the delay @rh-max! Merging.

**Pull request merged**